docs/startingguide.rst

```diff
@@ -0,0 +1,383 @@
+for those less familiar with Python, CartoPy, and matplotlib. To demonstrate the ease of
+creating a plot with this interface, let's make a color-filled plot of temperature using
+NARR data.
+
+.. code-block:: python
+
+    import xarray as xr
+    from metpy.cbook import get_test_data
+    from metpy.plots import ImagePlot, MapPanel, PanelContainer
+    from metpy.units import units
+
+    # Use sample NARR data for plotting
+    narr = xr.open_dataset(get_test_data('narr_example.nc', as_file_obj=False))
+
+    img = ImagePlot()
+    img.data = narr
+    img.field = 'Geopotential_height'
+    img.level = 850 * units.hPa
+
+    panel = MapPanel()
+    panel.area = 'us'
+    panel.layers = ['coastline', 'borders', 'states', 'rivers', 'ocean', 'land']
+    panel.title = 'NARR Example'
+    panel.plots = [img]
+
+    pc = PanelContainer()
+    pc.size = (10, 8)
+    pc.panels = [panel]
+    pc.show()
+
+.. image:: _static/declarative_example.png
+   :width: 800 px
+   :align: center
+
+Other plot types are available, including contouring to create overlay maps. For an example of
+this, check out the :doc:`Combined Plotting <examples/plots/Combined_plotting>` example. MetPy
+Monday videos `#69`_, `#70`_, and `#71`_ also demonstrate the declarative plotting interface.
+
+.. _`#69`: https://youtu.be/mbxE2ovXx9M
+.. _`#70`: https://youtu.be/QgS27jwj8OI
+.. _`#71`: https://youtu.be/RBJ8Pm7x4ok
+
+----------------------
+Other Python Resources
+----------------------
+
+While MetPy does a lot of things, it doesn't do everything. Here are some other good resources
+to use as you start using MetPy and Python for meteorology and atmospheric science:
+
+**Training and Example Sites**
+
+* `Online Python Training`_
+* `Unidata Python Gallery`_
+* `Unidata Python Workshop`_
+* `MetPy Monday Playlist`_
+
+**Useful Python Packages**
+
+* `Siphon`_: remote access of meteorological data via THREDDS servers
+* `Xarray`_: reading/writing labeled N-dimensional arrays
+* `Pandas`_: reading/writing tabular data
+* `NumPy`_: numerical computations
+* `Matplotlib`_: creation of publication-quality figures
+* `CartoPy`_: publication-quality cartographic maps
+* `SatPy`_: read and visualize satellite data
+* `PyART`_: read and visualize radar data
+
+.. _Siphon: https://unidata.github.io/siphon/
+.. _`Unidata Python Workshop`: https://unidata.github.io/python-workshop
+.. _`MetPy Monday Playlist`:
+     https://www.youtube.com/playlist?list=PLQut5OXpV-0ir4IdllSt1iEZKTwFBa7kO
+.. _`Xarray`: http://xarray.pydata.org/en/stable/
+.. _`Pandas`: https://pandas.pydata.org
+.. _`NumPy`: https://numpy.org/devdocs
+.. _`Matplotlib`: https://matplotlib.org
+.. _`CartoPy`: https://scitools.org.uk/cartopy/docs/latest/
+.. _`SatPy`: https://satpy.readthedocs.io/en/latest/
+.. _`PyART`: https://arm-doe.github.io/pyart/
+
+-------
+Support
+-------
+
+Get stuck trying to use MetPy with your data? Unidata's Python team is here to help! Email us
```

Comment:
Would it be possible to link instead to `SUPPORT.md`? This would keep from repeating this information everywhere.

Reply:
This would be great, but we currently don't publish the `SUPPORT.md` page to our docs (it's a level above). Currently we list all of our support links out in `index.rst`. We could just refactor all of this to move the support page into the docs itself.

Reply:
or we could cite it like we do in README and use ` https://github.com/Unidata/MetPy/blob/master/SUPPORT.md`

Reply:
Wait, we just list `CONTRIBUTING` in `index.rst`...how is that working?

Reply:
oh, I'm an idiot. `docs/` has a softlink to it.

Reply:
Updated with the html link

Reply:
Can you take a stab at including SUPPORT in our docs? I just think that might be nicer and less jarring than going from the one website to an entirely different site, different layout, etc. You should be able to just repeat what we're doing with CONTRIBUTING.

Reply:
Done. I agree that this is the better way to go.